cli/cmd/install_test.go

```diff
@@ -174,20 +174,51 @@ func TestValidate(t *testing.T) {
 		}
 	})
 
-	t.Run("Rejects invalid log level", func(t *testing.T) {
+	t.Run("Rejects invalid controller log level", func(t *testing.T) {
 		options := newInstallOptions()
 		options.controllerLogLevel = "super"
 		expected := "--controller-log-level must be one of: panic, fatal, error, warn, info, debug"
 
 		err := options.validate()
 		if err == nil {
-			t.Fatalf("Expected error, got nothing")
+			t.Fatal("Expected error, got nothing")
 		}
 		if err.Error() != expected {
 			t.Fatalf("Expected error string\"%s\", got \"%s\"", expected, err)
 		}
 	})
 
+	t.Run("Properly validates proxy log level", func(t *testing.T) {
```

Comment:
TIOLI: maybe the test name should be more consistent with the controller log level test? Like
```suggestion
	t.Run("Rejects invalid proxy log level", func(t *testing.T) {
```

Reply:
I called it like that because it no only rejects invalid values, but it also tests for valid values :grin: